content/guides/virtualizing/xen.md

```diff
@@ -5,11 +5,11 @@ date = "2019-12-7"
 tags = []
 +++
 
-Virtual instances of operating systems are perfect for all kinds of testing purposes that need to be done in a safe and isolated environment. Installing Haiku in a virtual machine is a solution for people who do not want to install it on their physical computers but want to become familiar with it.
+Virtual instances of operating systems are perfect for all kinds of testing purposes that need to be done in a safe and isolated environment. Therefore, installing Haiku in a virtual machine is an ideal solution for people who do not want to install it on their physical computers but want to become familiar with it.
 
-In this guide, the Haiku operating system is being run under virtual circumstances using Ubuntu Linux, Xen and Virt-Manager. but you can use any distribution of Linux that supports Xen.
+The guide was tested on an Ubuntu Linux installation, but it should work on any Linux distribution that supports Xen.
 
-This tutorial, for demonstration purposes, uses an Anyboot image. It can be obtained [here](/get-haiku). Both the ISO and Anyboot images are available there. Verify the checksums to make sure the downloaded files are not corrupted.
+This tutorial uses an Anyboot image for demonstration purposes. Both the ISO and the Anyboot image can be obtained [here](/get-haiku). Make sure to verify the checksums in order to make sure that your downloaded files are not corrupted.
```

Comment:
We only have anyboot images to download.

Reply:
I really can't remember why I wrote that in the first place, then.